.github/workflows/ci_cd.yaml

```
@@ -0,0 +1,78 @@
+        uses: docker/setup-buildx-action@v3.8.0
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ env.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Build Docker image with tags
+        run: >
+          docker build
+          -t ${{ env.DOCKER_REPO }}:latest 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)